web/subscribers/index.php

```diff
@@ -1,35 +1,46 @@
 <?php
 
+// File to use the required functions
 require_once("../functions.php");
 
-// Default parameters if not specified
-$defaults = [
-    "color" => "blue",
+// making the youtube Api Url to access the JSON data
+$apiUrl = "https://www.googleapis.com/youtube/v3/channels?";
+$apiUrl .= http_build_query(array(
+            "part" => "statistics",
+            "id"   => "UCipSxT7a3rn81vGLw9lqRkg",// your channel id
+            "alt"  => "json",
+            "key"  => "[YOUR API KEY HERE]" // you can put your API key here if you deploy it yourself
+                            ));
+//getting the data from the url as JSON
+$response = curl_get_contents($apiUrl);
+
+// decoding the json data hence we can extract the Subscribers count
+$decoded = json_decode($response);
+
+// getting the subscribers count by querying throught the decoded json
+$Subscribers = $decoded->items[0]->statistics->subscriberCount;
+
+// getting the subscribers count to a rounded value and right format
+$message = shortNumber($Subscribers);
+
+// parameters for the Shield.io url to get the label and message in a particular format
+$contents = [
+    "label" => "Youtube Subscribers",
+    "message" => $message,
+    "color" => "brightgreen",
     "logo" => "youtube",
     "logoColor" => "white",
-    "style" => "flat-square",
-    "format" => "short",
-    "label" => "YouTube subscribers",
+    "style" => "plastic",
     "labelColor" => "gray",
     "id" => "UCipSxT7a3rn81vGLw9lqRkg",
     "key" => "[YOUR API KEY HERE]" // you can put your API key here if you deploy it yourself
 ];
 
-// Query for finding subscriber count in JSON
-$query = "$.items[0].statistics.subscriberCount";
+// Build the Shields.io url using the above parameters
+$url = "https://img.shields.io/static/v1?".http_build_query($contents);
 
-// Build the Shields.io url using the above parameters and JSON query
-$url = getShieldURL($query, $defaults);
+//getting the contents of the image
+$image = curl_get_contents($url);
 
-// Set the content type for output
-header('Content-type: image/svg+xml');
-
-// Get response from the URL and output its contents
-$response = curl_get_contents($url);
-
-// get format param and format the subscriber count accordingly
-$format = validateParam("format", "/^(commas|short|none)$/", $defaults);
-$style = validateParam("style", "/^[A-Za-z\-]+$/", $defaults);
-$response = formatResponseNumber($response, $format, $style);
```

Comment:
The functionality of choosing the number format should not be removed. This should be done in the `getShieldsURL` function as well before creating the static shield url.

Reply:
Hey @DenverCoder1, I have made the required changes as you requested, please review them...